RELEASE_NOTES/3.66.md

```diff
@@ -0,0 +1,21 @@
+# CLI
+- Display daily upgrade reminder when a new version is available
+- Improve handling of 429 rate limiting for large projects
+
+# CLI-kit
+- Add cached package version checks
+- Fix small UI issue when using an invalid shop
+
+# App
```

Comment:
```suggestion
# App
- General availability of log streaming for Shopify Functions in the `app dev` and `app logs` commands
```

Reply:
Thanks, added here and in the GitHub release